README.md

```diff
@@ -1 +1,61 @@
-# eoslime-shape-react
\ No newline at end of file
+# eoslime-shape-react
+
+The project is a complete and ready to use dApp with predefined React front-end and EOSLime integration. It consists of an EOS smart contract, deployment script, unit tests and a web part. The dApp represents a simple todo list, showing how to read and record a data on the blockchain through the browser. In gives a helpful example and foundation for a further development of more sophisticated dApp/s.
+
+In order to get the project up and running you need to do the following steps:
+
+1. Install eoslime
```

Comment:
Add Prerequisites section

Reply:
Add in it following heading 3 sections

Installation
Compile contract
Deploy contract

Reply:
Installation
```
  # Install EOSLime globaly to use the CLI
    npm install -g eoslime
    
    # Fetch the project
    eoslime shape react
    
    # Install dependencies
    cd eoslime-shape-react
    npm install
```

Reply:
Compile contract

```
 # The command will read contract files from ./contracts folder
    # Output will be stored as .abi and .wasm files in ./compiled folder
    eoslime compile
```

Reply:
Deploy contract

[Nodeos Message image]

```
# The command will run deployment script from ./deployment folder
eoslime deploy
```

The script will do several things

- Will create a random account and will deploy the contract code on it
- Will create a random account for Alice. She will make transactions to the contract
- Both accounts will be uploaded in .config file located at **./web/src/config.json**

Reply:
updated